File: security-svc/src/test/java/com/chilly/security_svc/containers/ContainersEnvironment.java
package com.chilly.security_svc.containers;

import org.chilly.common.test.containers.EurekaTestContainer;
import org.chilly.common.test.containers.PostgresTestContainer;
import org.testcontainers.junit.jupiter.Container;
import org.testcontainers.junit.jupiter.Testcontainers;

@Testcontainers
public class ContainersEnvironment {
    @Container
    public static PostgresTestContainer postgresTestContainer = PostgresTestContainer.getInstance();
    @Container
    public static EurekaTestContainer eurekaContainer = EurekaTestContainer.getInstance();
}

File: places-svc/src/test/kotlin/com/chilly/places_svc/containers/ContainersEnvironment.kt
package com.chilly.places_svc.containers;

import org.chilly.common.test.containers.EurekaTestContainer
import org.chilly.common.test.containers.PostgisTestContainer
import org.testcontainers.junit.jupiter.Container
import org.testcontainers.junit.jupiter.Testcontainers

@Testcontainers
open class ContainersEnvironment {

    companion object {
        @Container
        val postgresTestContainer: PostgisTestContainer = PostgisTestContainer.getInstance()
        @Container
        val eurekaTestContainer: EurekaTestContainer = EurekaTestContainer.getInstance()
    }
}
